File: sprint2/C#/lesson5code1.cs
using System;

class Node {
    public string value;
    public Node next;
    
    public Node(string value = null, Node next = null) {
        this.value = value;
        this.next = next;
    }
}

class Program {
    static void PrintLinkedList(Node vertex) {
        while (vertex != null) {
            Console.Write($"{vertex.value} -> ");
            vertex = vertex.next;
        }
        Console.WriteLine("None");
    }
    
    static void Main() {
        Node n3 = new Node("third");
        Node n2 = new Node("second", n3);
        Node n1 = new Node("first", n2);
        PrintLinkedList(n1);
        PrintLinkedList(n2);
    }
}

File: sprint2/JS/lesson5code1.js
class Node {
  constructor(value = null, next = null) {
    this.value = value;
    this.next = next;
  }
}

function printLinkedList(vertex) {
  while (vertex) {
    process.stdout.write(`${vertex.value} -> `);
    vertex = vertex.next;
  }
  console.log('None');
}

const n3 = new Node('third');
const n2 = new Node('second', n3);
const n1 = new Node('first', n2);

printLinkedList(n1);
printLinkedList(n2);
